packages/gatsby-plugin-referrer/src/gatsby-browser.js

```diff
@@ -0,0 +1,8 @@
+exports.onClientEntry = function() {
+  if (document.cookie.indexOf(`gatsbyOriginalReferrer`) < 0) {
```

Comment:
just sanity check - we only care about totally first visit? even if someone manually entered url in browser?